mws/mws.py

```diff
@@ -1121,6 +1121,46 @@ def update_inbound_shipment(self, shipment_id, shipment_name,
             ))
         return self.make_request(data, method="POST")
 
+    def get_inbound_guidance_for_sku(self, skus=None, marketplace_id=None):
+        """
+        Returns inbound guidance for a list of items by Seller SKU
+        to help you decide if an item should be sent to a given marketplace.
+        """
+        marketplace_id = marketplace_id or 'ATVPDKIKX0DER'
```

Comment:
We probably shouldn't default the `marketplace_id` to 'ATVPDKIKX0DER' in the library as we don't do it elsewhere in the code

Reply:
Those lines are removed.